distribution/test/thirdparty/restito/src/main/java/com/xebialabs/restito/server/secure/SecureStubServer.java

```diff
@@ -0,0 +1,252 @@
+/**
```

Comment:
Submit a PR to Restito with these new changes and don't forget to document that this work around should be removed if the PR was successfully submitted.